test/integration/features/config.feature

```diff
@@ -74,7 +74,7 @@ Checks whether CRC `config set` command works as expected in conjunction with `c
         @darwin
         Examples:
             | property                             | value1 | value2 |
-            | skip-check-virtualbox-installed      | true   | false  |
+            | skip-check-hyperkit-installed        | true   | false  |
```

Comment:
@jsliacan this looks like some extra whitespace at the end of `skip-check-hyperkit-installed`, `warn-check-hyperkit-installed` and `hyperkit`

Reply:
I reformatted all tables in `config.feature`, but some of those you pointed out didn't change. It doesn't look like there are extra white spaces though if you look here (I use an auto-fmt for feature files in emacs): https://github.com/jsliacan/crc/blob/killvboxrefs/test/integration/features/config.feature

Reply:
The formatting is done with whitespaces, the ones which are highlighted in this diff is the 'ed' before which is changed to '  '. Change looks good to me, especially as CI passes ;)